_api-reference/cat/cat-tasks.md

```diff
@@ -15,20 +14,37 @@ redirect_from:
 
 The CAT tasks operation lists the progress of all tasks currently running on your cluster.
 
+<!-- spec_insert_start
+api: cat.tasks
+component: endpoints
+-->
 ## Endpoints
-
 ```json
-GET _cat/tasks
+GET /_cat/tasks
 ```
+<!-- spec_insert_end -->
 
-## Query parameters
 
-Parameter | Type | Description
-:--- | :--- | :---
-nodes | List | A comma-separated list of node IDs or names to limit the returned information. Use `_local` to return information from the node you're connecting to, specify the node name to get information from specific nodes, or keep the parameter empty to get information from all nodes.
-detailed | Boolean | Returns detailed task information. (Default: false)
-parent_task_id | String | Returns tasks with a specified parent task ID (node_id:task_number). Keep empty or set to -1 to return all.
-time | Time | Specify the units for time. For example, `5d` or `7h`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
+<!-- spec_insert_start
+api: cat.tasks
+component: query_parameters
+columns: Parameter,Type,Description,Default
+include_deprecated: false
+-->
+## Query parameters
+Parameter | Type | Description | Default
```

Comment:
Can we make this "Data type" to be consistent with the template?

Reply:
Sure. But I don't think this should be a blocker for this PR since @nhtruong would have to address it in the workflow.

Reply:
Addendum to my previous. I figured out how to update this and will add it to this PR.